rpm/nemo-qml-plugin-calendar-qt5.spec

```diff
@@ -26,6 +26,19 @@ Requires:   %{name} = %{version}-%{release}
 %description tests
 %{summary}.
 
+%package lightweight
+Summary:    Calendar lightweight QML plugin
+Group:      System/Libraries
+BuildRequires:  pkgconfig(Qt5Core)
```

Comment:
Subpackage is not built separately so the BuildRequires directives should be pointless.